src/traces/parcoords/base_plot.js

```diff
@@ -0,0 +1,80 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var d3 = require('d3');
+var Plots = require('../../plots/plots');
+var Registry = require('../../registry');
+var xmlnsNamespaces = require('../../constants/xmlns_namespaces');
+
+exports.name = 'parcoords';
+
+exports.attr = 'parcoords';
+
+exports.idRoot = 'parcoords';
+
+exports.idRegex = /^parcoords([2-9]|[1-9][0-9]+)?$/;
+
+exports.attrRegex = /^parcoords([2-9]|[1-9][0-9]+)?$/;
+
+exports.plot = function(gd) {
+    var Parcoords = Registry.getModule('parcoords');
+    var calcData = Plots.getSubplotCalcData(gd.calcdata, 'parcoords', void(0));
```

Comment:
A PR feedback suggested that I migrate to `getSubplotCalcData`, but maybe it's not a good fit or I'm doing it the wrong way?

Currently `parcoords` don't seem to mix with e.g. `scatter`, unlike e.g. the `scatter` - `scatter` [example](https://plot.ly/javascript/subplots/). Is it a goal that it be supported? I don't know right now what steps would be needed, so any info is useful.